src/platform-includes/session-replay/configure-sample-rate/javascript.mdx

```diff
@@ -0,0 +1,19 @@
+```javascript {tabTitle: ESM}
+// If you're using one of our integration packages, like `@sentry/angular`,
+// substitute its name for `@sentry/browser` here
+import * as Sentry from "@sentry/browser";
+
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This enables automatic instrumentation (highly recommended), but is not
+  // necessary for purely manual usage
```

Comment:
```suggestion
```

I would honestly just delete this to keep it as simple as possible for the majority of our users. (see my comment about "Advanced Configuration", as we're describing more manual usage in the Readme).